libs/base/src/lib/context/AccountProvider.tsx

```diff
@@ -249,8 +256,43 @@ const OnboardProvider: FC<{
         console.error(error)
       })
     }
+
+    fetch(`${API_ENDPOINT}/signature`)
```

Comment:
Does this work without an address passed?

Reply:
@chrisjgf yes... when we don't pass an address, we get the signature msg, I added some changes in the API to support this.

Reply:
Nice, just checking! 👍